services/services.rst

```diff
@@ -171,15 +99,17 @@ For accessing on-line, you will need to `install clients <https://zoom.us/downlo
 For scheduled list of upcoming teleconferences for PFS technical team, 
 the project is providing a list of scheduled teleconferences calendar at 
 `google calendar <https://calendar.google.com/calendar/embed?src=su0pbsaull17etlj62tet5anm0%40group.calendar.google.com>`_.
-
+Also `meeting reminder notification service <https://pfs.ipmu.jp/notification/>`_ 
+is provided to get reminders via phone, text (SMS), email, or push notification 
+to your web browser. 
 
 Services for software development at pfspipe.ipmu.jp and external
 ------
 
 PFS software development group has its dedicated website at 
 `pfspipe.ipmu.jp <https://pfspipe.ipmu.jp/>`_, and also uses several external 
 services like `GitHub Subaru-PFS organization <https://github.com/Subaru-PFS>`_. 
-Followings are a list of commonly used services, refer top page at 
+Followings are a list of frequently used services, refer top page at 
 `pfspipe.ipmu.jp <https://pfspipe.ipmu.jp/>`_ and links in the page 
```

Comment:
services, -> services. 
(comma -> period)

You deleted details of individual contents of pfs.ipmu.jp (https). Similarly, you could delete or shorten the below as a list??

Reply:
YES! thank you for catch. #8c4717c